components/nav/Horizontal.vue

```diff
@@ -71,22 +65,33 @@
         @click="app.drawer = true"
       />
     </template>
+
+    <template #extension>
+      <VProgressLinear
+        :indeterminate="app.loading"
+        height="1"
+        :color="app.loading ? 'accent' : 'background'"
+        bg-opacity="0"
+      ></VProgressLinear>
+    </template>
   </VAppBar>
 </template>
 
-<style scoped lang="scss">
+<style lang="scss">
```

Comment:
Would this not effect the styling of `VToolbar` used in other components, if we were to implement it at a later stage?

Reply:
You're right, fixed